src/Microsoft.AspNet.Http/Features/FormFeature.cs

```diff
@@ -119,7 +144,7 @@ public IFormCollection ReadForm()
                     var section = await multipartReader.ReadNextSectionAsync(cancellationToken);
                     while (section != null)
                     {
-                        var headers = new HeaderDictionary(section.Headers);
+                        var headers = section.Headers;
                         ContentDispositionHeaderValue contentDisposition;
                         ContentDispositionHeaderValue.TryParse(headers[HeaderNames.ContentDisposition], out contentDisposition);
```

Comment:
That wouldn't wrap: https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.AspNet.WebUtilities/MultipartSection.cs#L25-L38